### ISSUE TITLE
Where to put groups / teams / communities in directory structure?

### ISSUE THREAD

**Issue by tnabtaf** [galaxyproject/galaxy-hub#81]
This thread was inspired by a discussion about teams in issue #36, which is about where to put person pages, 

It may be useful to have a defined place (or places) in the Hub directory structure for doc about groups related to Galaxy.  

**There are several types of groups to consider**

1. Domain and task based groups
   * Galaxy-Proteomics and Galaxy-P
   * Natural Language Processing 
   * Intergalactic Utilities Commission (IUC): Tool wrapping, best practices, ToolShed.
   * Galaxy Training Network
   * GalaxyAdmins
   * Galaxy Committers
2. Geography and language based
   * Galaxy Australia, Galaxy UK, Galaxy Pitagora, Galaxy Arabic, Galaxy France & IFB Galaxy Working Group
3. Institutional
   * Freiburg Galaxy Team
   * Any university / research organization with a significant Galaxy presence

*I'm going to argue that case 3 is distinct from the first two, and that this issue should focus on just the first two cases*.  Case 3, Institutions, can (and should be?) dealt with in a separate directory structure.  If there is consensus on this split, we can create a separate issue for institutions.

## Naming / Where to put this info?

The old wiki put this info in the `Community` hierarchy. Some thoughts for the hub:

`/src/communities/`
`/src/groups/`
`/src/teams/`

I favor `/src/communities/` because, well, these are all communities, and I really like that word; or `/src/groups/` because these are all groups, and that's less typing.

## Metadata

I'd favor having the root page be largely driven by metadata on the subpages.  The root page might be a directory generated from the subpage metadata.  The subpages could contain metadata, plus any additional text specific to that community.

**Comment by hrhotz** [comment]
yes, it will definitively require a split between 1/2 and 3.   "3" is the replacement of "https://wiki.galaxyproject.org/Community/Deployments", isn't-it?

I would even argue, that it needs three different directory structure, ie splitting 1 and 2. I consider them pretty different as well.

**Comment by tnabtaf** [comment]
> "3" is the replacement of "https://wiki.galaxyproject.org/Community/Deployments", isn't-it?

I was thinking of folding that in somehow, although I don't think we should require organizations to describe their servers to be listed in this.

After the holidays, I'll put up an easy way to vote on whatever proposals show up here, including how to split these up.

**Comment by hrhotz** [comment]
maybe not "require" organizations to describe their servers, but leave the option in. So far, the old wiki page has been a good resource to point people to, who ask about hardware requirement/advice on the mailing lists
